CMakeLists.txt

```diff
@@ -10,10 +10,14 @@ set(GAMMA_INSTALL_BIN_DIR ${PROJECT_SOURCE_DIR}/bin)
 # Set paths for external dependencies
 set(STB_INCLUDE_DIR ${PROJECT_SOURCE_DIR}/extern/stb)
 set(GLM_INCLUDE_DIR ${PROJECT_SOURCE_DIR}/extern/glm)
```

Comment:
Can you also remove the GLM dependency? Otherwise I can do it as well

Reply:
fixed in  249f0b3